lib/librender.c

```diff
@@ -963,57 +1020,62 @@ int addClassificationToPolarVolume(PolarVolume_t* pvol, float ****tensor, int di
         }
         
         PolarScan_addParameter(scan, mistnetParamClassification);
+        RAVE_OBJECT_RELEASE(mistnetParamClassification);
+        RAVE_OBJECT_RELEASE(scan);
     }
     
     return(0);
 
 }
 
-
-#ifdef MISTNET
-
+#if defined(MISTNET)
```

Comment:
why here `#if defined()` and elsewhere `#ifdef`?

Reply:
No difference in behavior but changed it back to #ifdef MISTNET anyway. When using defined() you can add multiple conditions.